src/lib/tabmanager.js

```diff
@@ -206,6 +206,11 @@ TabManager.prototype = {
     return recentlyAddedGroup;
   },
 
+  getGroupTabCount: function(tabBrowser, groupID) {
+    let tabs = this._storage.getTabIndexesByGroup(tabBrowser, groupID);
+    return tabs.length;
```

Comment:
```js
return this._storage.getTabIndexesByGroup(tabBrowser, groupID).length;
```
No real need of initializing a variable here since the code is as readable if we return the result directly.